Batteries/Tactic/Lint/Frontend.lean

```diff
@@ -108,7 +109,7 @@ def lintCore (decls : Array Name) (linters : Array NamedLinter) :
         BaseIO.asTask do
           match ← withCurrHeartbeats (linter.test decl)
               |>.run' mkMetaContext
-              |>.run' {options, fileName := "", fileMap := default} {env}
+              |>.run' {options, fileName := "", fileMap := default, initHeartbeats } {env}
```

Comment:
It looks like the `withCurrHeartbeats` sets `initHeartbeats` from `IO.getNumHeartbeats`. What effect does initializing `initHeartbeats` here have? Can you come up with a test that validates this?

Reply:
I think you're right, this has no effect.